### ISSUE TITLE
[1.2-rc.14] Crash on code updates

### ISSUE THREAD

**Issue by raix** [meteor/meteor#5140]
I do `$ meteor run android`
Then update the code
Get this crash when the code is updating:
```bash
=> Started app on Android Device.             
                                              
/Users/raix/.meteor/packages/meteor-tool/.1.1.7-rc.13.hwrd8w++os.osx.x86_64+web.browser+web.cordova/mt-os.osx.x86_64/dev_bundle/lib/node_modules/fibers/future.js:245
						throw(ex);
						      ^
ReferenceError: plugins is not defined
    at /tools/runners/run-app.js:662:58
    at [object Object]._.extend._runOnce (/tools/runners/run-app.js:665:8)
    at [object Object]._.extend._fiber (/tools/runners/run-app.js:858:28)
    at /tools/runners/run-app.js:396:12
```

**Comment by haozhengfeng** [comment]
i also have  this  issue

**Comment by martijnwalraven** [comment]
Thanks for the report! This should be fixed in the next RC.

**Comment by Tim-W** [comment]
Same here. Runs perfectly fine first time, but have to re-run `meteor run android-device` everytime I do an update.

**Comment by stubailo** [comment]
Should be fixed in the newly released `1.2-rc.15` version! Let me know if it isn't.
